lightningd/invoice.c

```diff
@@ -82,8 +82,7 @@ static struct command_result *tell_waiter(struct command *cmd,
 		json_add_invoice(response, details);
 		return command_success(cmd, response);
 	} else {
-		/* FIXME: -2 should be a constant in jsonrpc_errors.h.  */
-		response = json_stream_fail(cmd, -2,
+		response = json_stream_fail(cmd, INVOICE_EXPIRED_DURING_WAIT,
```

Comment:
Probably also need to update: https://github.com/ElementsProject/lightning/blame/602b81fef2f47834aa815d723d0f210af80062ce/doc/lightning-waitinvoice.7.md#L24-L26

Reply:
Done in e3879d93.